mobile/lib/widgets/settings/networking_settings/external_network_preference.dart

```diff
@@ -0,0 +1,199 @@
+import 'dart:convert';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/extensions/build_context_extensions.dart';
+import 'package:immich_mobile/models/auth/auxilary_endpoint.model.dart';
+import 'package:immich_mobile/entities/store.entity.dart' as db_store;
+import 'package:immich_mobile/widgets/settings/networking_settings/endpoint_input.dart';
+
+class ExternalNetworkPreference extends HookConsumerWidget {
+  const ExternalNetworkPreference({super.key, required this.enabled});
+
+  final bool enabled;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final entries =
+        useState([AuxilaryEndpoint(url: '', status: AuxCheckStatus.unknown)]);
+    final canSave =
+        useState(entries.value.every((e) => e.status == AuxCheckStatus.valid));
+
+    saveEndpointList() {
+      canSave.value =
+          entries.value.every((e) => e.status == AuxCheckStatus.valid);
+
+      final endpointList = entries.value
+          .where((url) => url.status == AuxCheckStatus.valid)
```

Comment:
Shouldn't this prevent you from saving?

Reply:
If there's an invalid endpoint, shouldn't that be an error instead of silently ignoring it?

Reply:
There is a UI feedback when you entering the invalid endpoint